lib/definitions.php

```diff
@@ -210,32 +210,40 @@
 );
 
 //Helpdesk ticket priority
+define('RT_PRIORITY_IDLE', -3);
 define('RT_PRIORITY_VERYLOW', -2);
 define('RT_PRIORITY_LOW', -1);
 define('RT_PRIORITY_NORMAL', 0);
-define('RT_PRIORITY_URGENT', 1);
-define('RT_PRIORITY_CRITICAL', 2);
+define('RT_PRIORITY_HIGHER', 1);
+define('RT_PRIORITY_URGENT', 2);
+define('RT_PRIORITY_CRITICAL', 3);
 
 $RT_PRIORITIES = array(
+	RT_PRIORITY_IDLE => trans('idle'),
 	RT_PRIORITY_VERYLOW => trans('very low'),
 	RT_PRIORITY_LOW => trans('low'),
 	RT_PRIORITY_NORMAL => trans('normal'),
+	RT_PRIORITY_HIGHER => trans('higher'),
 	RT_PRIORITY_URGENT => trans('urgent'),
 	RT_PRIORITY_CRITICAL => trans('critical'),
 );
 
 $RT_PRIORITY_STYLES = array(
-	RT_PRIORITY_VERYLOW => 'background-color: blue; color: white;',
-	RT_PRIORITY_LOW => 'background-color: lightgreen; color: black;',
+	RT_PRIORITY_IDLE => 'background-color: transparent; color: gainsboro;',
+	RT_PRIORITY_VERYLOW => 'background-color: transparent; color: silver;',
+	RT_PRIORITY_LOW => 'background-color: transparent; color: grey;',
```

Comment:
Moim zdaniem źle dobrane kolory...

Reply:
O te 3? Czy któryś konkretnie?

Reply:
Dla mnie mogą być ostatnie dwa silver albo nawet wszystkie 3 poniżej normal jako silver.

Reply:
Bo one mają nieprzeszkadzać a color-coding ma być >=normal

Reply:
Tak - fajnie wyglądają wszystkie >normal jako silver. Co Ty na to?

Reply:
Niebieski rzuca się w oczy strasznie, tak samo jak zielony. Chcemy docelowo zwiekszyć czytelność color-coding.

Reply:
Zróbmy tak włącz webeditor i wybierz według Ciebie poprawniejsze. Po czym puść pull requesta. A ja sobie lokalnie ustawie szare na zaniżonych priorytetach.